### PR TITLE
Create rob-van-pamel.md

### PR DESCRIPTION
**Pull request by robvanpamel**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1043) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1043) before we can accept your contribution.<br/><br/>:x: robvanpamel [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1043)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=1043) it.</sub>

**Review by Arhell — APPROVED** [review]

**Comment by rprouse** [comment]
@robvanpamel you'll need to sign the CLA before we can merge this.

**Comment by robvanpamel** [comment]
> @robvanpamel you'll need to sign the CLA before we can merge this.

This is done now. Thanks

File: input/community/speakers/rob-van-pamel.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+Title: Rob Van Pamel
+Pronouns: MR
```
Comment (rprouse) [inline input/community/speakers/rob-van-pamel.md:3]: This is causing a build error, should be in the form he/him 

```
Error: [/home/runner/work/website/website/input/community/speakers/rob-van-pamel.md => community/speakers/rob-van-pamel.html] Preferred gender pronouns should be of the form [subject]/[object] in lowercase (I.e. "they/them") (ValidateSpeakerPronouns)
```

**Comment by isaacrlevin** [comment]
@robvanpamel please update pronoun

**Comment by isaacrlevin** [comment]
@robvanpamel closing this PR. Please update your branch and create a new one

**Pull request closed without merging**